tsconfig.base.json

```
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "target": "ES2015",
+    "module": "esnext",
+    "moduleResolution": "node",
+    "lib": ["ES2015", "dom"],
+    "sourceMap": true,
+    "strict": true,
+    "strictFunctionTypes": false,
+    "forceConsistentCasingInFileNames": true,
+    "esModuleInterop": true,
+    "skipDefaultLibCheck": true,
+    "skipLibCheck": true,
```

Per [docs](https://www.typescriptlang.org/tsconfig#skipDefaultLibCheck) `skipDefaultLibCheck` is deprecated, and I think it's supposed to do the same as `skipLibCheck`?
```suggestion
    "skipLibCheck": true,
```

---

interesting. probably should send PR to the docsite , cli explicitly says it's deprecated. thx